DB/src/main/java/io/deephaven/db/tables/utils/TableManagementTools.java

```diff
@@ -113,6 +132,7 @@ public static void writeTable(Table sourceTable, String destDir, StorageFormat s
     public static void writeTable(Table sourceTable, TableDefinition definition, File destDir, StorageFormat storageFormat) {
         if (storageFormat == StorageFormat.Parquet) {
             writeParquetTables(new Table[]{sourceTable}, definition, CompressionCodecName.SNAPPY, new File[]{destDir}, definition.getGroupingColumnNamesArray());
+            TableDefinition.persistDefinition(definition, destDir);
```

Comment:
As mentioned in our call, I think we shouldn't do this, and I think this means that we aren't testing the new schema conversion logic as intended.

Reply:
Done, no more writing of table def.